examples/with_airflow/with_airflow/airflow_kubernetes_dag.py

```diff
@@ -2,7 +2,7 @@
 
 from airflow import models
 from airflow.operators.dummy_operator import DummyOperator  # type: ignore
-from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import KubernetesPodOperator
+from airflow.providers.cncf.kubernetes.operators.pod import KubernetesPodOperator
```

Comment:
im surprised we are pinning and changing this

Reply:
pinned because there was some kind of pyright env dependency conflict, but the new path is supported in both older and newer versions, so it's future-proof